### PR TITLE
Made significant changes for production deployment

### PR DESCRIPTION
**Pull request by dntandan**

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -3,39 +3,34 @@ import React from 'react';
...
           </Link>
           <Link href="https://kutt.it/techhub-invite">
-            <li className="lg:text-lg text-sm leading-5 lg:mr-10 mr-2 flex flex-col items-end justify-end font-normal cursor-pointer hover:text-brand-600">
-              Discord
-            </li>
+            <a>
+              {' '}
+              <li className="lg:text-lg text-sm leading-5 lg:mr-10 mr-2 flex flex-col items-end justify-end font-normal cursor-pointer hover:text-brand-600">
+                Discord
+              </li>
```
Comment (adikeshri) [inline components/Footer.tsx:29]: Should we add contributors in footer too?

**Review by adikeshri — CHANGES_REQUESTED** [review]
There are few console.logs(contributors part), remove these before moving to PROD. Also if possible remove as much commented out code as possible and unused imports

File: components/Footer.tsx (hunk 1 above)
Comment (dntandan) [reply components/Footer.tsx:29]: > OkThere are few console.logs(contributors part), remove these before moving to PROD. Also if possible remove as much commented out code as possible and unused imports

Ok will do that

Comment (dntandan) [reply components/Footer.tsx:29]: > Should we add contributors in footer too?

Actually We should

**Review by adikeshri — CHANGES_REQUESTED** [review]
Move spinner to a common component, so we can import and reuse again. I think we should use spinner in contributions and last 10 question page too.

Let's just move it to a commom component for now

**Comment by dntandan** [comment]
Ok will do that also

**Review by adikeshri — APPROVED** [review]
LGTM

**Pull request merged**